### ISSUE TITLE
Question about Dataset

### ISSUE THREAD

**Issue by ihuachang** [giuliomattolin/ConfMix#8]
Thank you for your code. 

I have a question regarding your Adaptive Learning implementation. 
In this approach, both the source and target training data have annotation files in YOLO format. 
However, in unsupervised Domain Adaptation, shouldn't there be no annotation files for the target data? 
Please let me know if I have misunderstood any information. 

Thank you.

**Comment by roger0426** [comment]
Hi, have you figure out your problem? 
I got some errors about fog_train_dataset labels and also be confused when i was reproduce this work.

Thank you.

**Comment by ihuachang** [comment]
Hi, just use a dummy annotation file, for example, assign every image with a bounding box [0,0,0,0]. The program only needs this file to register the dataset; it won't be used in the validation or training part.
